source/renderer/app/components/layout/TopBar.scss

```diff
@@ -20,6 +20,10 @@
       text-align: center;
       text-shadow: 0 2.5px 10px rgba(0, 0, 0, 0.25);
       user-select: text;
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
```

Comment:
@yakovkaravelov please order CSS rules alphabetically 🙏

Reply:
Fixed all the css rules sorting.